utils.go

```diff
@@ -0,0 +1,25 @@
+package main
+
```

Comment:
could we change the file name to ping, or network, or whatever? Generally I don't like a "utils" file which contains all unrelated util funcs.